charts/drax/charts/service-monitor/values.yaml

```
@@ -22,6 +22,12 @@ image:
   pullPolicy: IfNotPresent
   tag: ""
 
+config:
+  namespaces:
+    - "default"
```

Maybe in the namespace we install drax in?

```suggestion
  namespaces:
    - "{{ .Release.Namespace }}"
```

It will also require a `tpl` call in the configmap to work.